docs/examples/apt-cacher-ng.md

```diff
@@ -80,6 +85,16 @@ which obey `http_proxy`, such as `curl`, `wget` and others:
 might need to do it and you can do it from your `Dockerfile`
 too.
 
+**Option 4** is to link Debian-containers to proxy server using following command:
+
+    $ docker run -i -t --link test_apt_cacher_ng:apt_proxy -e http_proxy=http://apt_proxy:3142/ debian bash
+
+**Option 5** is to create a custom network of APT proxy server and Debian-based containers:
```

Comment:
Sentence doesn't make sense.

    **Option 5** creates a custom network of APT proxy servers and Debian-based containers